### ISSUE TITLE
Does not detect package.json in Windows environment

### ISSUE THREAD

**Issue by b3rnhard** [JoshuaKGoldberg/eslint-plugin-package-json#13]
The culprit is the following line:

```
const isPackageJson = filePath =>
    filePath.endsWith('/package.json') || filePath === 'package.json';
```

filePath in my case is `C:\Users\bla\workspace\project\package.json` and does not match due to the `/` in your `endsWith()` test.
